securedrop/source_templates/index.html

```diff
@@ -62,7 +62,7 @@
     Included here so the images can preload while the user is first
     reading the page. Hidden by default. -->
     <div class="bubble">
-      <p>You appear to be using the Tor Browser. You can turn the Security Slider to High in 4 easy steps!</p>
+      <p>You appear to be using the Tor Browser. In Tor's "Security Settings" pane (accessible from the toolbar's onion-button), we reccommend turning Security Slider to High—it's easy-peasy!</p>
```

Comment:
That's ok! Instructions are often redundant. The point of the edit, is that currently users have to mouse-over something to access that ordered-list... and they shouldn't have to. The single line of text should explain all they need to know, to directly act. But if they STILL don't get it, then they have the ordered list. Standard pattern. 

The less stuff users have to access to know what to do, the better. :D